### PR TITLE
(PE-12458) File path changes for PE client tools

### PR DESCRIPTION
**Pull request by jdwelch**
Update to the file_path.md file to capture recent additions for PE client tools

### CODE REVIEW

File: file_paths.md [hunk 1]
```diff
@@ -161,6 +165,11 @@ create a `puppet` user or group.
     Path                                      Setting                        3.x
     C:\ProgramData                                                           n/a
 
+    C:\ProgramData\PuppetLabs/client-tools    # default client tool settings
```
Comment (shrug) [inline file_paths.md]: There's a '/' where there should be a '\'

Do we need a config dir here? like the "C:\ProgramData\PuppetLabs\puppet\etc" below? I'm not sure what else might land in the C:\ProgramData\PuppetLabs\client-tools directory

Comment (jdwelch) [inline file_paths.md]: Ah, i'll fix the copypasta error. I'm not sure what else would end up there, but presuming there is another tool every release, and we don't do the work to consolidate all client configs into one file (which would be the goal, but here we are), there would be the 1:1 correspondence between tool and config file.

**Comment by MosesMendoza** [comment]
+1

**Comment by jdwelch** [comment]
Can one of @shrug @MosesMendoza merge this if you're happy with it?

**Comment by shrug** [comment]
ping @joshcooper @kylog Do y'all have any input here? I'll merge otherwise

**Comment by kylog** [comment]
:+1:

**Comment by joshcooper** [comment]
LGTM. One side note, the directory `C:\ProgramData\PuppetLabs\client-tools` is writable by all users, and the MSI should restrict permissions on that directory as it will presumably contain passwords, etc, e.g. https://github.com/puppetlabs/puppet_for_the_win/blob/master/wix/puppet.wxs#L296-L301. It's not hard to do, but we need to make sure that happens. /cc @Iristyle

**Pull request merged**